synapseclient/__init__.py

```diff
@@ -1,6 +1,6 @@
-import importlib.resources
 import json
 
+import importlib_resources
```

Comment:
This was the library we didn't want to use right?

Reply:
😅

Reply:
fixed